media-gfx/pencil/pencil-0.6.4.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop l10n qmake-utils xdg-utils
```

Comment:
You should drop l10n.eclass and install all locales by default, this is now on user to INSTALL_MASK these unwanted small files. I don't really mind it, but it's the right thing to do.